serve/mlc_serve/engine/model_module.py

```diff
@@ -117,8 +117,8 @@ class TextGenerator(Protocol):
 
     def generate(
         self,
-        requests: List[Union[PrefillRequest, DecodeRequest]],
-        kv_cache: KVCache,
+        requests: Sequence[Union[PrefillRequest, DecodeRequest]],
```

Comment:
Why do we need these two lines of changes?

Reply:
See https://github.com/octoml/mlc-llm/pull/159#discussion_r1449359486